data/scripts/suite.py

```diff
@@ -139,9 +140,21 @@ def run_suite(args):
     parser.add_argument("--valgrind", help="Path to valgrind, if provided " \
                         "the tests are run with it", type=str)
     parser.add_argument("--valgrind-supp", help="Path to valgrind's suppression file", type=str)
+    parser.add_argument("--color", dest="color", help="Disable colored output", action="store_true")
+    parser.add_argument("--no-color", dest="color", help="Force enable colored output", action="store_false")
+    parser.set_defaults(color=None)
 
     args = parser.parse_args()
     if args.valgrind_supp:
         args.valgrind_supp = "--suppressions=%s" % args.valgrind_supp
+    if args.color is None:
+        args.color = sys.stdout.isatty()
```

Comment:
For my use cases (mostly keeping logs in the bots output) I don't need to check TERM. It's easy to add later inside this conditional.